sklearn/tests/test_discriminant_analysis.py

```diff
@@ -94,6 +94,24 @@ def test_lda_predict():
     assert_raises(ValueError, clf.fit, X, y)
 
 
+def test_lda_predict_proba():
+    # Test LDA posterior probabilities
+    # Binary case
+    clf = LinearDiscriminantAnalysis(solver='svd').fit(X6, y6)
+    probas = clf.predict_proba(X6)
+    assert_array_almost_equal(
+        probas[0],
+        np.array([0.91942108, 0.08057892])
+    )
+    # Multiclass case
+    clf = LinearDiscriminantAnalysis(solver='svd').fit(X6, y7)
+    probas = clf.predict_proba(X6)
+    assert_array_almost_equal(
+        probas[0],
+        np.array([0.25128617, 0.36876296, 0.37995087])
+    )
+
+
```

Comment:
Could you add a test to check that the argmax of the probability is giving the same than the `predict` function.

Reply:
We should also check that sum over the axis 1 is one.